2019-08-10 17:24:12,053 INFO --------------------------------
2019-08-10 17:24:12,054 INFO ------STARTING PHOTOBOOTH-------
2019-08-10 17:24:12,054 INFO --------------------------------
2019-08-10 17:24:12,054 INFO sys.path=[['/home/pi/bin', '/usr/lib/python2.7', '/usr/lib/python2.7/plat-arm-linux-gnueabihf', '/usr/lib/python2.7/lib-tk', '/usr/lib/python2.7/lib-old', '/usr/lib/python2.7/lib-dynload', '/usr/local/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages/PILcompat', '/home/pi/git/photobooth', '/home/pi/git/photobooth/scripts', '/home/pi/git/photobooth/coregui', '/home/pi/git/photobooth/corecam', '/home/pi/git/photobooth/uploaders', '/home/pi/git/photobooth/auth']]
2019-08-10 17:24:14,456 INFO In startBooth()...
2019-08-10 17:24:19,331 INFO Calling CountdownText()...
2019-08-10 17:24:21,374 INFO Back from CountdownText()
2019-08-10 17:24:23,381 INFO In BoothCamera.capture(): resolution= 3280x2464, ISO=200, sharpness=60
2019-08-10 17:24:24,464 INFO In archiveImage()...
2019-08-10 17:24:24,465 INFO archive_dir: /home/pi/git/photobooth/images
2019-08-10 17:24:24,466 INFO image_filename.ext: picture.jpg
2019-08-10 17:24:24,467 INFO next_idx: 25139148
2019-08-10 17:24:24,468 INFO new_filename: /home/pi/git/photobooth/images/picture_25139148.jpg
2019-08-10 17:24:24,541 INFO In archiveImage(): Exiting...
2019-08-10 17:24:24,585 INFO In displayImage():...
2019-08-10 17:24:24,830 INFO Exiting displayImage()
2019-08-10 17:24:24,831 INFO Upload to Picasa OFF in configuration
2019-08-10 17:24:24,831 INFO Uploading to Google Drive...
2019-08-10 17:24:24,832 INFO GoogleDriveUploader constructor: IN
2019-08-10 17:24:24,833 INFO BackgroundProcessor.run(): Awake now. Number of data sets in 'data_queue' = [0]
2019-08-10 17:24:24,833 INFO Authenticator: __init__(): client_secrets=[client_secrets.json], pickle_filename=[saved_creds.pickle]
2019-08-10 17:24:24,833 INFO BackgroundProcessor.run(): Nothing to upload; going to get some zzzzs
2019-08-10 17:24:24,834 INFO Authenticator:__init__(): Found existing pickled file in [/home/pi/git/photobooth/secrets/]
2019-08-10 17:24:24,836 INFO Authenticator:__init__(): Loaded existing pickle file into object
2019-08-10 17:24:24,837 INFO Authenticator:__init__(): Returning!
2019-08-10 17:24:24,842 WARNING file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/googleapiclient/discovery_cache/__init__.py", line 41, in autodetect
    from . import file_cache
  File "/usr/local/lib/python2.7/dist-packages/googleapiclient/discovery_cache/file_cache.py", line 41, in <module>
    'file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth')
ImportError: file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth
2019-08-10 17:24:24,843 INFO URL being requested: GET https://www.googleapis.com/discovery/v1/apis/drive/v3/rest
2019-08-10 17:24:25,057 INFO In BackgroundProcessor.kickOff()
2019-08-10 17:24:25,058 INFO GoogleDriveUploader.preWorkFunction()
2019-08-10 17:24:25,058 INFO BackgroundProcessor.enqueue(): Number of data sets in 'data_queue' = [1]
2019-08-10 17:24:25,059 INFO In BackgroundProcessors.kickOff() - Exiting.
2019-08-10 17:24:25,059 INFO BackgroundProcessor.run(): Awake now. Number of data sets in 'data_queue' = [1]
2019-08-10 17:24:25,059 INFO GoogleDriveUploader.workFunction()...
2019-08-10 17:24:25,130 INFO URL being requested: GET https://www.googleapis.com/drive/v3/files?q=mimeType%3D%27application%2Fvnd.google-apps.folder%27&fields=nextPageToken%2C+files%28id%2C+name%29&alt=json&spaces=drive
2019-08-10 17:24:26,109 INFO GoogleDriveUploader.workFunction(): Uploading to folder: [NY-2018-2019], thefile: [/home/pi/git/photobooth/images/picture_25139148.jpg]...
2019-08-10 17:24:26,637 INFO URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?fields=id&alt=json&uploadType=multipart
2019-08-10 17:24:30,067 INFO Exiting startBooth()
2019-08-10 17:24:30,988 INFO GoogleDriveUploader.workFunction(): DONE with [/home/pi/git/photobooth/images/picture_25139148.jpg]
2019-08-10 17:24:30,990 INFO BackgroundProcessor.run(): Awake now. Number of data sets in 'data_queue' = [0]
2019-08-10 17:24:30,991 INFO BackgroundProcessor.run(): Nothing to upload; going to get some zzzzs
2019-08-10 17:24:31,462 INFO onClose(): Cleaning up uploaders...
2019-08-10 17:24:31,463 INFO BackgroundProcessor.enqueue(): Number of data sets in 'data_queue' = [1]
2019-08-10 17:24:31,465 INFO BackgroundProcessor.run(): Awake now. Number of data sets in 'data_queue' = [1]
2019-08-10 17:24:31,465 INFO Terminating thread...
